raiden-ts/src/helpers.ts

```diff
@@ -507,7 +505,7 @@ export async function getState(
     log,
   }: Pick<RaidenEpicDeps, 'address' | 'network' | 'contractsInfo' | 'log'>,
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  storage: { state?: any; storage?: Storage } & RaidenDatabaseOptions = {},
+  storage: { state?: any; adapter?: any, prefix?: string },
```

Comment:
Build is failing because you removed the default assignment here

Reply:
Also, the type has a comma where it required a semicolon:
```suggestion
  storage: { state?: any; adapter?: any; prefix?: string },
```

Reply:
:see_no_evil: 
Something is wrong with me these days.

Reply:
:heavy_check_mark: